javascript/javascript3/week1/weather/weatherApp.js

```diff
@@ -0,0 +1,56 @@
+console.log("hej module");
+const conferfmBtn = document.querySelector("#confirm-city");
+const cityInput = document.querySelector("#city-name");
+const body = document.querySelector("body");
+let cityName;
+let nameOfCity = document.createElement("p");
+let temprature = document.createElement("p");
+let icon = document.createElement("p");
+let windSpeed = document.createElement("p");
+let clouds = document.createElement("p");
+let warningParagraph = document.createElement("p");
+let sunRiseAndSet = document.createElement("p");
+
+conferfmBtn.addEventListener("click", () => {
+  if (cityInput.value == "") {
+    warningParagraph.innerHTML = "Please enter a city name";
+    body.appendChild(warningParagraph);
+  } else {
+    async function weatherApp() {
```

Comment:
This function is better declared outside the scope of the click listener for better readability.